nautobot/extras/templates/extras/job_list.html

```diff
@@ -38,7 +38,7 @@ <h1>{% block title %}Jobs{% endblock %}</h1>
                                         <label class="label label-default">Read-only</label>
                                     {% endif %}
                                 </td>
-                                <td>{{ job.description | placeholder }}</td>
+                                <td>{{ job.description_first_line | render_markdown | striptags | placeholder }}</td>
```

Comment:
`striptags` will also remove things like bold, italics, and hyperlinks from the description, right? I feel like those are useful to retain, even if it does means that someone could mark the first line of their job description as an H1 and have it render as such -- I feel like that falls into a "Yes you can do that, but you shouldn't" case rather than something we need to go out of our way to block.

Reply:
OK cool, fair points.